app/components/ui/Datatable.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Datatable extends React.Component {
```

Comment:
This component is mostly pointless, but should allow us to improve table sorting/rendering functionality consistently down the line.